modules/appnexusAstBidAdapter.js

```diff
@@ -201,6 +201,7 @@ function newBid(serverBid, rtbBid) {
       image: nativeAd.main_img && nativeAd.main_img.url,
       icon: nativeAd.icon && nativeAd.icon.url,
       clickUrl: nativeAd.link.url,
+      clickTrackers: nativeAd.link.click_trackers,
```

Comment:
I wonder if we should be validating the different bid types and objects... food for later thought.